__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -12815,17 +12815,3 @@ Object {
   ],
 }
 `;
-
-exports[`#validate should colorize errors when \`opts.colorizeErrors\` is present 1`] = `
```

Comment:
This got removed because we're skipping this test and Jest didn't carry it over when it moved the snapshot. If we ever re-enable this test this snapshot will come back then.